docs/standard/commandline/snippets/customize-help/csharp/Program.cs

```diff
@@ -0,0 +1,183 @@
+            "--light-mode",
+            description: "Determines whether the background color will be black or white");
+        var foregroundColorOption = new Option<ConsoleColor>(
+            "--color",
+            description: "Specifies the foreground color of console output",
+            getDefaultValue: () => ConsoleColor.White);
+
+        var rootCommand = new RootCommand("Read a file")
+        {
+            fileOption,
+            lightModeOption,
+            foregroundColorOption
+        };
+
+        rootCommand.SetHandler(
+            (FileInfo file, bool lightMode, ConsoleColor color) =>
+            {
+                Console.BackgroundColor = lightMode ? ConsoleColor.White: ConsoleColor.Black;
+                Console.ForegroundColor = color;
+                Console.WriteLine($"--file = {file?.FullName}");
+                Console.WriteLine($"File contents:\n{file?.OpenText().ReadToEnd()}");
+            },
+            fileOption,
+            lightModeOption,
+            foregroundColorOption);
+
+        await rootCommand.InvokeAsync(args);
+        // </original>
+        Console.WriteLine("Default help");
+        await rootCommand.InvokeAsync("--light-mode false --color Yellow");
+        await rootCommand.InvokeAsync("-h");
+    }
+
+    static async Task First2Columns(string[] args)
+    {
+        var fileOption = new Option<FileInfo>(
+            "--file",
+            description: "The file to print out.",
+            getDefaultValue: () => new FileInfo("scl.runtimeconfig.json"));
+        var lightModeOption = new Option<bool>(
+            "--light-mode",
+            description: "Determines whether the background color will be black or white",
+            getDefaultValue: () => true);
+        var foregroundColorOption = new Option<ConsoleColor>(
+            "--color",
+            description: "Specifies the foreground color of console output",
+            getDefaultValue: () => ConsoleColor.White);
+
+        var rootCommand = new RootCommand("Read a file")
+        {
+            fileOption,
+            lightModeOption,
+            foregroundColorOption
+        };
+
+        rootCommand.SetHandler(
+            (FileInfo file, bool lightMode, ConsoleColor color) =>
+            {
+                Console.BackgroundColor = lightMode ? ConsoleColor.Black : ConsoleColor.White;
+                Console.ForegroundColor = color;
+                Console.WriteLine($"--file = {file?.FullName}");
+                Console.WriteLine($"File contents:\n{file?.OpenText().ReadToEnd()}");
+            },
+            fileOption,
+            lightModeOption,
+            foregroundColorOption);
+
+        // <first2columns>
+        var parser = new CommandLineBuilder(rootCommand)
+                .UseDefaults()
+                .UseHelp(ctx =>
+                {
+                    ctx.HelpBuilder.CustomizeSymbol(fileOption,
+                        firstColumnText: "--file <FILEPATH>");
```

Comment:
I really like the second example, but I believe this could be done with the options ArgumentDisplayName (not sure I am remembering that name precisely.)